sdks/python/apache_beam/io/filebasedsink_test.py

```diff
@@ -76,9 +75,9 @@ def _create_temp_file(self, name='', suffix=''):
 
 class MyFileBasedSink(filebasedsink.FileBasedSink):
 
-  @unittest.skipIf(sys.version_info[0] == 3, 'This test still needs to be '
-                                             'fixed on Python 3'
-                                             'TODO: BEAM-5627')
+  @unittest.skipIf(os.environ.get('RUN_SKIPPED_PY3_TESTS') != '1',
```

Comment:
This condition will skip the test under Python 2 as well.
Let's use:
 `@unittest.skipIf(sys.version_info[0] == 3 and not os.environ.get('RUN_SKIPPED_PY3_TESTS'))`